common/headers/version_checker.go

```diff
@@ -75,6 +76,7 @@ var (
 		ClientNameCLI:           "<2.0.0",
 		ClientNameServer:        "<2.0.0",
 		ClientNameUI:            "<3.0.0",
+		ClientNameNexusGoSDK:    "<1.0.0",
```

Comment:
Let's support v1 already in anticipation of a stable release.

```suggestion
		ClientNameNexusGoSDK:    "<2.0.0",
```